avocado/core/settings.py

```diff
@@ -30,26 +30,6 @@
 
 from ..utils import path
 
-if 'VIRTUAL_ENV' in os.environ:
-    CFG_DIR = os.path.join(os.environ['VIRTUAL_ENV'], 'etc')
-    USER_DIR = os.environ['VIRTUAL_ENV']
-else:
-    CFG_DIR = '/etc'
-    USER_DIR = os.path.expanduser("~")
-
-_config_dir_system = os.path.join(CFG_DIR, 'avocado')
-_config_dir_system_extra = os.path.join(CFG_DIR, 'avocado', 'conf.d')
-_config_dir_local = os.path.join(USER_DIR, '.config', 'avocado')
-_source_tree_root = os.path.join(sys.modules[__name__].__file__, "..", "..", "..")
-_config_path_intree = os.path.join(os.path.abspath(_source_tree_root),
-                                   'avocado', 'etc', 'avocado')
-_config_path_intree_extra = os.path.join(_config_path_intree, 'conf.d')
-
-config_filename = 'avocado.conf'
-config_path_system = os.path.join(_config_dir_system, config_filename)
-config_path_local = os.path.join(_config_dir_local, config_filename)
-config_path_intree = os.path.join(_config_path_intree, config_filename)
```

Comment:
Good, I double-checked and it looks like they really were only used in the __init__ so this is way better.